docs/source/blog/datashuttle.md

```diff
@@ -0,0 +1,220 @@
+---
+blogpost: true
+date: 25 April 2024
+author: Joe Ziminski, Niko Sirmpilatze
+location: London, UK
+category: Blog
+language: English
+image: 1
+---
+
+
+
+# Managing neuroscience projects with **datashuttle**
+*Create, validate and transfer standardised project folders*
+
+```{image} /_static/blog_images/datashuttle/datashuttle-overview-dark.png
+:align: center
+:class: only-dark
+:width: 650px
+```
+```{image} /_static/blog_images/datashuttle/datashuttle-overview-light.png
+:align: center
+:class: only-light
+:width: 650px
+```
+<br>
+
+Maintaining a well-organised neuroscience project is hard. 
+
+Although everyone can appreciate the benefits of a tidy project
+folder, the practicalities of running an experiment often gets 
+in the way. Folder organisation 
+is low on the list of priorities when it
+comes to acquisition sessions spent managing complex systems 
+and experimental animals.
+
+However, the cost of small mistakes during data acquisition can be high.
+One misplaced character may mean sessions are missed by analysis 
+scripts or subject identifications duplicated.
+The best protection against such errors is automating the process
+through acquisition scripts—resulting in hours spent writing code 
+managing the naming and transfer of folders—a task entirely 
+unrelated to the central research goals.
+
+In our previous blog, we highlighted the benefits of data standardisation 
+for systems neuroscience, introducing the 
+[NeuroBlueprint](https://neuroblueprint.neuroinformatics.dev/) 
+specification. 
+An immediate benefit of a widely-used standard is that the entire community
+can contribute to shared tools for project management.
+This means individual researchers don't waste time
+writing data-management code. 
+
+In this blog post we introduce 
+[**datashuttle**](https://datashuttle.neuroinformatics.dev/)—a 
+tool for the automated creation, 
+validation and transfer of projects organised to 
+the **NeuroBlueprint** standard. **datashutte** aims to
+drop into existing acquisition pipelines, reducing errors
+associated with manual folder creation and removing the need
+to write your down data-management code.
```

Comment:
What does this mean?

Reply:
Hmm I think this is an example of the below `awkwardness of this blogpost is it is trying to cater to both those who do data management manually as well as automated.`

It is basically saying three things at once:
- datashuttle should just drop into your existing pipeline at the stage of creating folders, and transferring them at the end of the session
- if you are creating your folders manually, it will help reduce typographical errors
- if you are writing your down data manangement code, it means you no longer have to.

Reply:
This sentence makes sense, I think I must have commented on the wrong line!

Reply:
I like how you have made the point here in these 3 bullet points. Maybe that's how we should argue it?

Reply:
>This sentence makes sense, I think I must have commented on the wrong line!

Oh I think it must be the "**down** data-management code." instead of own 😅

Reply:
how about:

```
And that really is all there is to **datashuttle**, which aims to, 
drop into your existing acquisition pipelines whether they be manual
or automated:
- the **graphical interface** can be used instead of manual folder creation, 
reducing the risk of typographical errors
- alternatively, the **Python API** can be used directly in automated acquisition 
pipelines, removing the need to write you own data-management code

Below we will give an overview of **datashttle**'s key folder creation
and transfer features.
```

(also introducing the two ways of running it here).

Reply:
Good to introduce the two ways of running it, but that first sentence is kinda weird (and has dodgy grammar). 

How about:

>*datashuttle**, aims to drop into your existing acquisition pipelines whether they be manual
or automated, with two ways to use it:
>- The **graphical user interface** can be used instead of manual folder creation, 
reducing the risk of typographical errors
>- The **Python API** can be used directly in automated acquisition 
pipelines, removing the need to write you own data-management code

>Below we will give an overview of **datashttle**'s key folder creation
and transfer features.

Reply:
Great thanks!